kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -284,7 +284,8 @@
         if (option === Options.REMOVE) {
           this.facilityForRemoval = facility;
         } else if (option === Options.REGISTER) {
-          this.facilityForRegister = facility;
+          this.$router.push({ path: '/manage' });
```

Comment:
We should add an additional `else if` here for handling the sync schedule option, rather than overwriting the register option.